lib/src/RedisChannel.cpp

```diff
@@ -175,7 +175,7 @@ sai_status_t RedisChannel::wait(
 
         swss::Selectable *sel;
 
-        int result = s.select(&sel, REDIS_ASIC_STATE_COMMAND_GETRESPONSE_TIMEOUT_MS);
+        int result = s.select(&sel, REDIS_ASIC_STATE_COMMAND_GETRESPONSE_TIMEOUT_MS * 10);
```

Comment:
please not use * 10

Reply:
This was not ideal approach, should have changed context, but this was debug code - at one point, we timed out. Just tried some experiments with current default of 6000 ms, seems ok, so will remove.

Reply:
currently timeout is 60 seconds, and you increases this to 10 minutes, please remove

Reply:
Not sure I understand, did I not address this already?

https://github.com/sydlogan/sonic-sairedis/commit/8695a23e13596d4f7bec1f2b7ce40745f54dc921